stanovy.tex

```diff
@@ -172,10 +178,14 @@ \section{Členská schůze}
     je oprávněn účastnit se zasedání a požadovat i dostat na něm vysvětlení 
     záležitostí Spolku, vztahuje-li se požadované vysvětlení k předmětu zasedání
     členské schůze.
+    \item Členové Spolku se mohou členské schůze účastnit osobně, distančně,
+    mohou předat své hlasovací právo, nebo se svého hlasovacího práva pro
+    konkrétní členskou schůzi vzdát. Jejich rozhodnutí ohledně způsobu účasti
+    na členské schůzi musí být ověřitelná a přítomná v zápise.
     \item Každý aktivní člen má jeden hlas, hlasy všech aktivních členů mají 
     stejnou váhu. Členská schůze je schopna usnášet se za účasti nadpoloviční 
-    většiny všech aktivních členů Spolku. Usnesení členská schůze přijímá 
-    většinou hlasů přítomných aktivních členů. 
+    většiny všech aktivních členů Spolku, kteří se nevzdali svého hlasu pro tuto schůzi.
+    Usnesení členská schůze přijímá většinou hlasů aktivních členů účastnících se této schůze. 
```

Comment:
To znamená, že pokud někomu předám hlas, pak se schůze účastním... ale nejsem přítomen. (Vyměnil jsem slovíčka z toho důvodu, aby se to špatně neinterpretovalo.)

Reply:
Len zdoraznujem, ze tu je zasadna zmena: Bude sa hlasovat vacasinou na schodzi, nie vacsinou clenov celkovo.

Reply:
Ahoj, vážně to chceme? Myslím, že v kontextu aktuálního fungování Spolku to bude fungovat dobře, dlouhodobě mi ale hlasování většinou *přítomných* přijde nebezpečné.

Dávám ke zvážení, zda-li neiplementovat hlasování stejně, jako to mají spolky v občanském zákoníku:
 * hlasuje se *většinou na schůzi*
 * avšak *schůze musí být usnášeníschopná*, tzn. musí být přítomna nadpoloviční většina členů (v našem případě bych uvážil do počtu přítomných i ty, kteří se účastní distančně a odečetl ty, kteří se hlasu explicitně vzdali)
* pokud není svhůze usnášeníschopná, do 14 dnů je svolána nová, která je usnášeníschopná vždy

Reply:
Zase je pravda, že z mé zkušenosti z oficiálních schůzí to fakt často dopadá tak, že se potká pár lidí na té první schůzi, zvolí si termín té schůze náhradní a až tam se fakticky něco děje :/.

Reply:
Hlasování dle Honzova návrhu mi přijde sympatičtější.

Reply:
Jsem z této diskuze zmaten... Ten diff říká `většiny všech aktivních členů Spolku, kteří se nevzdali svého hlasu pro tuto schůzi.`... tedy celá diskuze je nevalidní, protože diskutuje hlasování většinou přítomných, který ale není na pořadu dne.

Víceméně je zde malá změna oproti dnešku... lidé se mohou pro schůzi vzdát hlasu (sami dobrovolně)... Stále ale platí, že hlasují všichni aktivní členové! 

Navíc jsme se rozhodli problém ještě zmenšit tím, že aktivní člen může předat své hlasovací právo, nebo se může účastnit distančně... A v neposlední řadě také tím, že výbor může z neaktivní odebírat.

@horacekj jaký je tedy problém?

Reply:
@repli2dev To je aj moja chyba, s @horacekj sme neprecitali nove znenie uplne (GitHub totiz nad komentarom zobrazuje len par riadkov). Stav stanov je viac-menej taky, ako Honza navrhuje (tj. uznasaniaschopnost od polovice aktivnych clenov, hlasuje polovica pritomnych, nie je tam ale poznamka o dalsej schodzi, ktorou by som to nekomplikoval), takze tato konverzacia je irelevantna. Tj. zatvaram ju.